lib/src/native/include/dart/dart_api_dl.h

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
Move to `third_party/`. (Actually for this code it does not really matter because we also release this repo under Dart project authors copyright, but it's a good thing to always put third party code in `third_party/`.)